codec-dns/pom.xml

```diff
@@ -53,6 +53,16 @@
       <artifactId>netty-codec</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
I would prefer to make this an extra maven module to not add more dependencies here.